bin/autojump

```diff
@@ -135,10 +135,13 @@ def add_path(data, path, weight=10):
     with resulting duplicate entries in the database than a single canonical
     path.
     """
-    path = unico(path).rstrip(os.sep)
+    path = os.path.abspath(path)
```

Comment:
Gotcha. I changed it to this:
path = os.path.abspath(unico(path))
I tested in python 2.6 and it has the same effect as this (outputting a unicode string)
path = unico(os.path.abspath(path))
From what I can tell the path methods match their output encoding to the input, so as long as we input a unicode string we can expect one out.